harness/determined/common/experimental/user.py

```diff
@@ -118,3 +119,56 @@ def _from_bindings(cls, user_bindings: bindings.v1User, session: api.Session) ->
         user = cls(session=session, user_id=user_bindings.id)
         user._hydrate(user_bindings)
         return user
+
+
+class TokenType(enum.Enum):
+    # UNSPECIFIED is internal to the bound API and is not be exposed to the front end
+    USER_SESSION = bindings.v1TokenType.USER_SESSION.value
+    LONG_LIVED_TOKEN = bindings.v1TokenType.LONG_LIVED_TOKEN.value
+
+
+class UserSessions:
+    """
+    A class representing a UserSessions object that contains user session token info and
+    long lived access token info.
+
+    It can be obtained from :func:`determined.experimental.client.list_tokens`
+
+    Attributes:
+        session: HTTP request session.
+        id: (int) The ID of the user session.
+        user_id: (int) Unique ID for the user.
+        expiry: (Mutable, Optional[str]) Timestamp expires at reported.
+        created_at: (Mutable, Optional[str]) Timestamp created at reported.
+        token_type: (Mutable, Optional[TokenType]) Token type of the token.
+        is_revoked: (Mutable, Optional[bool]) Whether the token is active or revoked.
+        token_description: (Mutable, Optional[str]) Human-friendly description of token.
+    """
+
+    def __init__(self, token_id: int, session: api.Session):
+        self.id = token_id
+        self._session = session
+
+        self.user_id: Optional[int] = None
+        self.expiry: Optional[str] = None
+        self.created_at: Optional[str] = None
+        self.token_type: TokenType = TokenType.LONG_LIVED_TOKEN
```

Comment:
I am not sure about line 154 & 163 currently as the below alternative throws the error:

`self.token_type = userSession.token_type.value`

`ERROR: Item "None" of "Optional[v1TokenType]" has no attribute "value"`